src/components/idleUsers/card/card.module.scss

```diff
@@ -25,9 +25,13 @@ $theme-pink: #e30464;
   }
 
   .name {
-    margin-top: 3%;
-    margin-bottom: 0;
+    display: flex;
     font-size: 1rem;
     font-weight: 700;
+    width: 10rem;
+    text-align: center;
+    height: 1.5rem;
```

Comment:
This is a good approach but can cause problems when the username is too big. 
I generally use `min-height` property to set a default height to the element.

Reply:
But that would get the height of card to increase if we get the user's last name in new line when the first name is long